readme.md

```diff
@@ -211,6 +211,7 @@
 - [Deno](https://github.com/denolib/awesome-deno#readme) - A secure runtime for JavaScript and TypeScript that uses V8 and is built in Rust.
 - [DOS](https://github.com/balintkissdev/awesome-dos#readme) - Operating system for x86-based personal computers that was popular during the 1980s and early 1990s.
 - [Nix](https://github.com/nix-community/awesome-nix#readme) - Package manager for Linux and other Unix systems that makes package management reliable and reproducible.
+- [Integration](https://github.com/stn1slv/awesome-integration#readme) - Application and data integration.
```

Comment:
The description should describe what "integration" is in this context. Same at the top of your Awesome readme.

Reply:
@sindresorhus thanks for the review, I have added the context for the integration.

Reply:
@sindresorhus as for the new description, I pushed it in f4b52c3. Please let me know any other issues to resolve.